### DISCUSSION TITLE
Loot Lookup Idea

### DISCUSSION THREAD

**Discussion by Deathlysmiles** [runelite/runelite#16110] [Ideas]
Instead of typing in a monster would it be possible to make it so you type in an item and it displays all the monsters that drop said item along with the drop chances?

**Comment by Nightfirecat** [comment]
Loot tracker has nothing to do with drop chances. It sounds to me like you're interested in using the [Loot Lookup plugin from the hub](https://runelite.net/plugin-hub/show/loot-lookup).

**Reply by Deathlysmiles** [reply]
That's my bad, that's actually what I meant. I want like a reverse engineered version of Loot lookup so that it looks up items and give monster drops instead of looking up monsters for their drops.
